bugsnag/wsgi/middleware.py

```diff
@@ -1,8 +1,17 @@
 from webob import Request
+import sys
 
 import bugsnag
 from bugsnag.wsgi import request_path
 
+bottle_present = False
+if 'bottle' in sys.modules:
+    bottle_present = True
+    try:
+        import bottle
```

Comment:
I wonder if this could give false positives, just because `bottle` is importable does not mean the code base is using it.

Bottle could be installed as a global dependency on a server (perhaps on a shared web hosted, such as `apt-get install python-bottle`), there may be other Python applications using bugsnag-python running on that machine which would incorrectly report they are using bottle of a given version. I am wondering more about shared hosting providers.

Reply:
I think that was what line 8 (`if 'bottle' in sys.modules`) is supposed to partially account for (if the user imported bottle, then they are likely using it).

Reply:
Indeed, I believe this will only trigger if bottle is imported elsewhere.